distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -130,6 +118,26 @@ public void testCswRegistryUpdate() throws Exception {
     }
 
     @Test
+    public void testCswRegistryUpdateFailure() throws Exception {
+        String id = createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51280");
+
+        given().body(Library.getCswRegistryUpdate()
```

Comment:
Do you have any suggestions on alternatives to use for `replace` and `replaceAll`?